com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/cubrid/CUBRIDSQLHelper.java

```diff
@@ -176,6 +178,35 @@ public String getDBQualifier(String objectName) {
         return getQuotedObjName(objectName);
     }
 
+    /**
+     * If the referential action is 'SET NULL', check the column and change it to 'RESTRICT' if it
+     * is 'NOT NULL'
+     *
+     * @param referentialAction int
+     * @param fk FK
+     * @return int
+     */
+    private int changeReferentialAction(int referentialAction, FK fk) {
+        if (referentialAction != 2) {
```

Comment:
I think it would be better to use defined variables.

Reply:
Thank you for your review.
It is the same as the answer in the review below.